<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Photos</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frmMain">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <widget class="QFrame" name="frmSource">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTreeView" name="trvwSource"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frmDestination">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTreeView" name="trvwDestination"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmButtons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>408</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImport">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Import</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>759</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImport</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1031</x>
     <y>759</y>
    </hint>
    <hint type="destinationlabel">
     <x>623</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trvwSource</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>ImportDialog</receiver>
   <slot>onSourceDirClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trvwDestination</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>ImportDialog</receiver>
   <slot>onDestinationDirClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>1048</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSourceDirClicked(QModelIndex)</slot>
  <slot>onDestinationDirClicked(QModelIndex)</slot>
 </slots>
</ui>
